virttest/utils_misc.py

```diff
@@ -2298,7 +2298,7 @@ def get_qemu_best_cpu_model(params):
     return params.get("default_cpu_model", None)
 
 
-def get_qemu_version(params):
+def get_qemu_version(params=None):
```

Comment:
set params={ } as default

Reply:
pls refer to https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html as @lolyu mentioned above